### ISSUE TITLE
Arbitrary view after successful login

### ISSUE THREAD

**Issue by huiyie** [huiyie/hacktober-learn-react#4]

**Comment by huiyie** [comment]
Imagine you are an admin. After signing in, you want to manage ordinary users. Create a view to display and edit users' id, username and notes (like comments).

**Comment by abdulamite** [comment]
Can I take this one?

**Comment by huiyie** [comment]
yup sure, it will be interesting, looking forward to seeing this! 👍
